src/dxtbx/model/detector.py

```diff
@@ -774,13 +775,26 @@ def imgCIF_H(cbf_handle, sensor):
         size = tuple(reversed(cbf_handle.get_image_size(0)))
 
         try:
-            undefined_value = find_undefined_value(cbf_handle)
+            underload_value = find_underload_value(cbf_handle)
+        except Exception:
+            underload_value = None
+
+        try:
+            undefined_value = find_undefined_value(cbf_handle) + 1
+        except Exception:
+            undefined_value = 0
+
+        try:
             # In imgCIF overload means the pixel is saturated and hence untrusted
             # https://www.iucr.org/__data/iucr/cifdic_html/2/cif_img.dic/Iarray_intensities.overload.html
-            overload = cbf_handle.get_overload(0)
-            trusted_range = (undefined_value + 1, (overload - 1) * dxtbx_overload_scale)
+            overload = (cbf_handle.get_overload(0) - 1) * dxtbx_overload_scale
         except Exception:
-            trusted_range = (0.0, 1.0e6)
+            overload = 1.0e6
```

Comment:
This makes me wonder is there a specific reason for this particualr fallback value?

Reply:
Good question!  Some digging shows it used to be 0, 0 which was not so useful, so @graeme-winter changed it in eb05fbc5fcb630f48ad724fd742f09f907f31cda.

Reply:
No specific reason why 1e6: it is a generic big number